cms/djangoapps/contentstore/views/course.py

```diff
@@ -642,6 +642,8 @@ def _create_or_rerun_course(request):
             fields['display_name'] = display_name
 
         if 'source_course_key' in request.json:
+            fields['advertised_start'] = None
```

Comment:
@AkA84 , this definitely does the job for this bug, but it also feels oddly specific. @doctoryes , do you know if we have an existing way to make some settings not get duplicated over course runs?